code/Aggregate.m

```diff
@@ -0,0 +1,25 @@
+function Aggregate(genefile,Data,outdir)
+load(genefile);
+
+Str = Data;
```

Comment:
Is all of this the same as the corresponding removed code from `SCINGE.m` or did some parts change?

Reply:
This is the same as the corresponding removed code.